File: controllers/TestCase Controllers/Java/app/code.java
import java.util.Scanner;

public class ArraySum {
    public static void main(String[] args) {
        int[] numbers = new int[4]; // Create an array of size 4
        int sum = 0;
        
        Scanner scanner = new Scanner(System.in);
       
        // Read user input and store it in the array
        for (int i = 0; i < 4; i++) {
            System.out.print("");
            numbers[i] = scanner.nextInt();
        }

        // Calculate the sum of the numbers in the array
        for (int i = 0; i < 4; i++) {
            sum += numbers[i];
        }

        // Display the sum
        System.out.println(sum);
        
        // Close the scanner
        scanner.close();
    }
}

File: controllers/TestCase Controllers/Java/app/code.py
def main():
    try:
        # Get user input for the array elements
        array = []
        for i in range(4):
            element = int(input())
            array.append(element)

        # Calculate the sum of array elements
        array_sum = sum(array)

        print(array_sum)
    except ValueError:
        print("Invalid input. Please enter valid numbers.")

if __name__ == "__main__":
    main()
